src/python/WMCore/WMSpec/Steps/Executors/LogArchive.py

```diff
@@ -61,6 +61,16 @@ def execute(self, emulator=None, **overrides):
         if hasattr(self.step, 'override'):
             overrides = self.step.override.dictionary_()
 
+        #TODO hard coded - should we get it from config (something like override above
+        eosStageOutParams = {}
+        eosStageOutParams['command'] = "xrdcp"
+        eosStageOutParams['option'] = "--wma-disablewriterecovery"
+        #TODO need to gt the correct prefix and upl
+        eosStageOutParams['phedex-node'] = "T2_CH_CERN"
+        eosStageOutParams['lfn-prefix'] = "root://eoscms.cern.ch//eos/cms"
```

Comment:
We need to find the right place? Should this different from fermi agent to cern agent?

Reply:
We currently are proposing:

```
root://eoscms.cern.ch//eos/cms/store/logs/prod/recent
```

This should be identical across the agents - but it'd be better to be able to override in an agent configuration file (where `None` indicates "do not stageout").  This way, we can quickly fix any issues post-deploy.

Reply:
Thanks Brian, I will make it configurable then.

Reply:
This will change once every few years, so unless it's takes less than 5min, I think it's completely fine to have it hardcoded. In addition to that, we can't make quick fixes at this stage because it goes inside the workflow sandbox (unless we patch those as well).

Reply:
I think it is relatively easy if we use existing override parameter. (Although I think it is ever set anywhere) I think we should probably set as None first, then turn on for few workflow initially and then make the default value as above, what do you think?

Reply:
How about we leave it as it is, then we find a namespace we can use for tests - just to make sure the whole thing works - then in the end we set it to None and leave the correct configuration commented out waiting for the final green light from CERN IT?

Reply:
That would be good. Thanks Alan.